language/tools/vm-genesis/src/lib.rs

```diff
@@ -539,7 +586,19 @@ fn reconfigure(
     gas_schedule: &CostTable,
     interpreter_context: &mut TransactionExecutionContext,
 ) {
-    let txn_data = TransactionMetadata::default();
+    let mut txn_data = TransactionMetadata::default();
+    txn_data.sender = account_config::association_address();
+
+    move_vm
+        .execute_function(
+            &LIBRA_TIME_MODULE,
+            &INITIALIZE,
+            &gas_schedule,
+            interpreter_context,
+            &txn_data,
+            vec![],
+        )
+        .expect("Failure reconfiguring the system");
 
     move_vm
```

Comment:
It looks like we `reconfigure` twice--once here, and once earlier at line 162. Do we need to  do this twice?